source/gui/build_appimage.sh

```diff
@@ -0,0 +1,97 @@
+#! /bin/sh
```

Comment:
Have you tested with various `sh` versions? E.g. on Debian, `sh` is not necessarily `bash`.

It's usually safer to state explicitly `/bin/bash`, unless you tested it with various `sh`-compatible shells.

Reply:
`bash` is not available in some minimal Linuxes, so it is a little avoided for Docker users. ^^'